tests/devices/qubit/test_simulate.py

```diff
@@ -1178,3 +1179,78 @@ def test_qinfo_tf(self):
 
         grad5 = grad_tape.jacobian(results[5], phi)
         assert qml.math.allclose(grad5, expected_grads[5])
+
+
+ml_frameworks_list = [
+    "numpy",
+    pytest.param("autograd", marks=pytest.mark.autograd),
+    pytest.param("jax", marks=pytest.mark.jax),
+    pytest.param("torch", marks=pytest.mark.torch),
+    pytest.param("tensorflow", marks=pytest.mark.tf),
+]
+
+
+# pylint:disable=too-few-public-methods
+@pytest.mark.unit
+class TestMidCircuitMeasurements:
```

Comment:
How did we check the rate of fisher p_value failures for these tests?

Reply:
Do you mean false positives? I guess the rate is going to be 0 because we fixed a seed.